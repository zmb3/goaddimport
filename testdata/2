// Package main is a test.
package main

func main() {
}
