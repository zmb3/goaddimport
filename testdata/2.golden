// Package main is a test.
package main

import "github.com/someuser/somepkg"

func main() {
}
